_posts/2019-06-18-list-strategy.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: posts
+title: "Automatic execution strategy selection in Bazel 0.27"
+authors:
+  - ishikhman
+---
+**tl;dr**: Exciting news! After [0.27](https://blog.bazel.build/2019/06/17/bazel-0.27.0.html) release Bazel can *auto select* a suitable execution strategy, 
+eliminating the need for manual configuration via command line flags in most cases. Customization and strategy enforcement is still possible if needed and has become much easier to configure.
+
+When Bazel executes commands that are a part of the build, such as compiler and linker invocations, test runs and so on, 
```

Comment:
Get rid of one of "so on"s (I know it came from my text, sorry about that!)

Reply:
'etc.' instead ? :)

Reply:
etc. works